diff --git a/node_modules/eth-revert-reason/index.js b/node_modules/eth-revert-reason/index.js
index 4e43d12..17f5cb9 100644
--- a/node_modules/eth-revert-reason/index.js
+++ b/node_modules/eth-revert-reason/index.js
@@ -16,7 +16,9 @@ async function getRevertReason (txHash, network = 'mainnet', blockNumber = undef
   await validateInputPostProvider(txHash, network, blockNumber, provider)
 
   try {
-    const tx = await provider.getTransaction(txHash)
+    const tx = await provider.getTransaction(txHash);
+    delete tx.maxPriorityFeePerGas;
+    delete tx.maxFeePerGas;
     const code = await getCode(tx, network, blockNumber, provider)
     return decodeMessage(code, network)
   } catch (err) {
